### DISCUSSION TITLE
Mac Version

### DISCUSSION THREAD

**Discussion by jahrends** [sn4k3/UVtools#288] [Q&A] [answered]
Is there a version that works on Mac?  I'm on Big Sur and it still just immediately crashes.

**Comment by sn4k3** [comment, marked as answer]
mac version is here: https://github.com/sn4k3/UVtools/releases/latest

**Reply by jahrends** [reply]
ystem Integrity Protection: enabled

Crashed Thread:        0  Dispatch queue: com.apple.main-thread

Exception Type:        EXC_CRASH (SIGABRT)
Exception Codes:       0x0000000000000000, 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

Application Specific Information:
abort() called

Thread 0 Crashed:: Dispatch queue: com.apple.main-thread
0   libsystem_kernel.dylib        	0x00007fff2041d92e __pthread_kill + 10
1   libsystem_pthread.dylib       	0x00007fff2044c5bd pthread_kill + 263
2   libsystem_c.dylib             	0x00007fff203a1406 abort + 125
3   libcoreclr.dylib              	0x00000001046ff8ae PROCAbort + 30
4   libcoreclr.dylib              	0x00000001042d25f7 PROCEndProcess(void*, unsigned int, int) + 215
5   libcoreclr.dylib              	0x000000010470235c UnwindManagedExceptionPass1(PAL_SEHException&, _CONTEXT*) (.cold.1) + 44
6   libcoreclr.dylib              	0x0000000104521946 UnwindManagedExceptionPass1(PAL_SEHException&, _CONTEXT*) + 822
7   libcoreclr.dylib              	0x00000001045219d6 DispatchManagedException(PAL_SEHException&, bool) + 134
8   libcoreclr.dylib              	0x0000000104485244 IL_Throw(Object*) + 548

**Reply by sn4k3** [reply]
which UVtools package are you using/downloaded?
macos is X64 or ARM64?

**Reply by sn4k3** [reply]
It's working fine on my fresh bigsur VM

![vmware_2021-09-09_17-49-38](https://user-images.githubusercontent.com/113281/132728601-2c3046fc-3d9c-4cf1-9f6a-c4cac4b6ba7e.png)

**Reply by jahrends** [reply]
That is because I must be an idiot... I had tried a few versions... I blame the security...  I blew away the old version and downloaded 21 again and now it comes up.  Just shoot me.  Trying to solve an elephants foot problem... ;)

**Reply by sn4k3** [reply]
There was a bug on older releases (Some ago) that prevent uvtools from open.
Also you need to allow it on security center for the first time
